components/pointsbox.tsx

```diff
@@ -10,13 +9,12 @@ interface Pointsboxprop {
 }
 
 const Pointsbox: React.FC<Pointsboxprop> = (props: Pointsboxprop) => {
-  const options: Option[] = [];
+  const options: Option[] = [{value: 0, label: '0 p'}];
```

Comment:
why cant the list be empty initially?

Reply:
Because we need the option to give 0 points. 
Although another solution could have been to start the for-loop on 0.